src/app/feed/feed-news/feed-news.component.html

```diff
@@ -1,3 +1,11 @@
+<div class="message">
+  <!-- Hook up a message if needed like google does "About xxxxx results (y.yy seconds)" -->
+  News tab shows results filtered according to news organizations. If you wish to propose or change this list, please check
```

Comment:
Put this message below `About xxxx results` . Lets follow a design similar to market leader.

Reply:
Made the changes.